packages/backend/src/nest/connections-manager/connections-manager.service.ts

```diff
@@ -511,11 +502,35 @@ export class ConnectionsManagerService extends EventEmitter implements OnModuleI
         await this.registrationService.registerOwnerCertificate(args)
       }
     )
+    // TODO: Save community CA data in LevelDB. Perhaps save the
+    // entire Community type in LevelDB. We can probably do this once
+    // when creating the community.
+    this.socketService.on(
+      SocketActionTypes.SEND_COMMUNITY_CA_DATA,
+      async (payload: PermsData) => {
+        this.logger(`socketService - ${SocketActionTypes.SEND_COMMUNITY_CA_DATA}`)
+        this.registrationService.permsData = payload
+      }
+    )
```

Comment:
Here I renamed LAUNCH_REGISTRAR to SEND_COMMUNITY_CA_DATA and also the corresponding saga. Other than that it's just deleting things. It might be a good idea to split the rename into a separate PR, but I think this is fine too.